### ISSUE TITLE
Navigation - final review touch ups

### ISSUE THREAD

**Issue by anthonydillon** [canonical-web-and-design/jaas-dashboard#898]
This task is to review all navigation in whole once all the piece land and build a list of small fixes here.

**Comment by long-chung** [comment]
Can we make this are not fixed? as it takes up a large portion of the screen area
![IMG_06E107A40E46-1](https://user-images.githubusercontent.com/68847062/109782469-d88e9800-7c00-11eb-91de-edac40612a2e.jpeg)

**Comment by anthonydillon** [comment]
@long-chung For sure :+1:

**Comment by long-chung** [comment]
*Mobile - Number of models missing
*Mobile - Increase bottom margin under search bar 
<img width="346" alt="Screenshot 2021-03-04 at 13 58 00" src="https://user-images.githubusercontent.com/68847062/109974875-ebcd6080-7cf1-11eb-90b3-4e588441acc3.png">
*Mobile - When you click a tab that is slightly off page; the link should be in view similar to a carousel behaviour
<img width="308" alt="Screenshot 2021-03-04 at 14 08 51" src="https://user-images.githubusercontent.com/68847062/109976128-5501a380-7cf3-11eb-85d6-80da4b127fba.png">
*Mobile - Could we truncate or ellipses the breadcrumb if it can't fit on one line?
<img width="233" alt="Screenshot 2021-03-04 at 14 10 37" src="https://user-images.githubusercontent.com/68847062/109976346-92fec780-7cf3-11eb-9650-2f1e81e0e12e.png">

*Desktop - Text alignment of model name and tabs is off
<img width="534" alt="Screenshot 2021-03-04 at 14 00 08" src="https://user-images.githubusercontent.com/68847062/109974999-0c95b600-7cf2-11eb-820f-947830177199.png">
*Desktop - Move tabs over to the right to allow for longer model names
*Desktop - Left margin of model count is 24px whereas position of model name in tabs and breadcrumb is 32px. Please make them all 24px
*Desktop - Left panel opens and closes when switching pages, this shouldn't happen.

**Comment by ziheliu214** [comment]
I think @long-chung  covered most of the points

for mobile and desktop

- [ ] model name on the top left should be always clickable and linking back to Model > application page.

**Comment by anthonydillon** [comment]
> *Mobile - Could we truncate or ellipses the breadcrumb if it can't fit on one line?

@long-chung this one is tricky and leads to other considerations such as what direction should we ellipse? How do you access the hidden navigation items? etc.

>  model name on the top left should be always clickable and linking back to Model > application page.

@ziheliu214 I am not sure what you mean. The only model name that is not a link is when you are on the model details. Or do you mean that should be clickable as another way to switch to the apps view for that model?

**Comment by long-chung** [comment]
@anthonydillon here's some examplea of what I was thinking
https://atlassian.design/components/breadcrumbs/examples
https://jsfiddle.net/noh78j2p/3/

**Comment by anthonydillon** [comment]
@long-chung yeah I see. I think that is slightly out of scope for this round. Would you mind creating an issue for this one and we could create a component that can be reused.

**Comment by anthonydillon** [comment]
@ziheliu214 done you ask

**Comment by long-chung** [comment]
@anthonydillon sure thing. I'll create a new issue for the breadcrumb
